docs/content/documentation/systems/Kernels/framework/CoupledTimeDerivative.md

```diff
@@ -1,7 +1,31 @@
 <!-- MOOSE Documentation Stub: Remove this when content is added. -->
 
 # CoupledTimeDerivative
-!syntax description /Kernels/CoupledTimeDerivative
```

Comment:
Why remove those?

Reply:
Well, you could argue that I needlessly duplicated my effort but the class description and the information I filed under `## Description` are generally redundant. If we're fine with the redundancy, I'll put them back in.